wrappers/python/xraydb/xraydb.py

```diff
@@ -430,8 +430,29 @@ def xray_lines(self, element, initial_level=None, excitation_energy=None):
                                                r.initial_level, r.final_level)
         return out
 
+    def xray_line_strengths(self, element):
+        """
+        return the absolute line strength in cm^2/gr for all available lines
+
+        arguments
+        ---------
+        element:  atomic number, atomic symbol for element
+
+        """
+        out = {}
+        for label, eline in self.xray_lines(element).items():
+            try:
+                edge = self.xray_edge(element, eline.initial_level)
+                mu = self.mu_elam(element, [edge.absorption_edge-2,
+                                            edge.absorption_edge+2], kind='photo')
+                out[label] = (mu[1]-mu[0]) * eline.intensity * edge.fluorescence_yield
+            except:
```

Comment:
yeah, that's fair.  I think that what happens here is that 'edge' may have some `None` values (as you hit the M lines, say).  But explicitly checking for that is probably better.